### PR TITLE
Add lodash to test matrix

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

**Comment by samccone** [comment]
ok looks like 8 real failures here, fairly sure they are all related to deep object equality and lodash setting some prop that is causing the diff to fail. any insight @megawac ?

File: test/ci.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (megawac) [inline test/ci.sh]: Cheater xD

Comment (jmeas) [inline test/ci.sh]: rofl

Comment (samccone) [inline test/ci.sh]: lol inlining all the code in the yml file is not really scalable

Comment (jmeas) [inline test/ci.sh]: true true

Comment (jasonLaster) [inline test/ci.sh]: but.... yaml was designed for inlined codes guyz

Comment (samccone) [inline test/ci.sh]: :D

**Pull request merged**